tests/unit/utils/tasks.spec.js

```diff
@@ -90,4 +90,57 @@ describe('tasks', () => {
       })
     })
   })
+  describe('dtMean', () => {
+    it('should format seconds to nice isodatetime format', () => {
+      const tests = [
+        {
+          taskNode: { node: null },
+          expected: undefined
+        },
```

Comment:
Should test the 0 case too
```suggestion
        },
        {
          taskNode: {
            task: {
              meanElapsedTime: 0
            }
          },
          expected: undefined
        },
```